pkg/controller/v1beta1/inferenceservice/reconcilers/raw/raw_kube_reconciler.go

```diff
@@ -60,12 +62,16 @@ func NewRawKubeReconciler(client client.Client,
 	if err != nil {
 		return nil, err
 	}
+	var multiNodeEnabled bool
+	if workerComponentMeta.Name != "" {
```

Comment:
better to check nil for the `workerComponentMeta` instead of the value of the name, I assume you can pass nil when there is no worker